src/emqtt_bench.erl

```diff
@@ -274,7 +274,7 @@ start(PubSub, Opts) ->
                           proc_lib:spawn(?MODULE, run, [self(), PubSub, WOpts])
                   end, lists:seq(1, NoWorkers)),
     timer:send_interval(1000, stats),
-    main_loop(os:timestamp(), 1+proplists:get_value(startnumber, Opts)).
```

Comment:
nit: starting from 1 is a bit confusing, better to start from 0 and print after `+1`